lerna-debug.log

```diff
@@ -0,0 +1,24 @@
+0 silly argv {
```

Comment:
don't commit this file

Reply:
Where did that come from?

Reply:
you had some problem with lerna, if it's not in the gitignore file, it should be